packages/http-client/src/common-client-options.ts

```diff
@@ -0,0 +1,90 @@
+import { JSONSchemaType } from "@typespec/compiler";
+
+export interface ClientEmitterOptions {
+  "api-version"?: string;
+  "examples-dir"?: string;
+  "package-version"?: string;
+  "package-name"?: string;
+  namespace?: string;
+  license?: {
+    name: string;
+    company?: string;
+    link?: string;
+    header?: string;
+    description?: string;
+  };
+}
+
+export const ClientEmitterOptionsSchema: JSONSchemaType<ClientEmitterOptions> = {
+  type: "object",
+  additionalProperties: false,
+  properties: {
```

Comment:
wondering if we should do that already in `http-client` we are now giving those as option that every http client (not even ours) are supposed to be  suppporting.

Reply:
Yeah makes sense, keeping in JS emitter for now and will discuss with the crew

Reply:
and I would ideally like that we get to #1297(options as typespec) as the json schema approach is messy